### PR TITLE
Requested changes in verif-store

### PR DESCRIPTION
**Pull request by SoujanyaPonnapalli**
This PR adds gRPC format, removes the tx->storage communication as requested in the review (PR #1) 
Also this PR; continues to work with:
getBlockByNumber => Block;
getBlockByHash => Block;

### CODE REVIEW

**Comment by SoujanyaPonnapalli** [comment]
Under discussion:

`getBlockByNumber` => `blockHash` or `Block`,
`getBlockByHash` => `Block` or `Block`[ ]

**Comment by amytai** [comment]
@SoujanyaPonnapalli could you actually do 2 things:
1) Add the service / gRPC stuff as a commit in your original PR #1 
2) Please remove the java file. I believe that's a generated file? We shouldn't commit those since they will undoubtedly cause unnecessary diffs.

**Comment by amytai** [comment]
BTW, feel free to take a look at PR #3 for how to do gRPC format -- in particular, you don't need to define a service per message type.

File: verif-store.proto [hunk 1]
```diff
@@ -2,62 +2,44 @@ syntax = "proto3";
...
 }
 
-/*
-** Storage nodes should be given a list of key value pairs
-** to insert and a list of keys to delete in one batch operation
-*/
-message kvpairMsg {
-  bytes key = 1;
-  bytes value = 2;
+service getBlockByNumber {
```
Comment (no2chem) [inline verif-store.proto]: I don't think you need to define a service per message?

File: verif-store.proto [hunk 2]
```diff
@@ -66,35 +48,32 @@ message getReturnMsg {
...
 */
-message getBlockByMinerInputMsg {
-  bytes minerId = 1;
+message getRecentMinerInputMsg {
 }
 
-message getBlockReturnMsg {
-  blockMsg block = 1;
+message getRecentMinerReturnMsg {
+  bytes coibase = 1;
```
Comment (no2chem) [inline verif-store.proto]: coibase -> coinBase

File: verif-store.proto [hunk 3]
```diff
@@ -66,35 +48,32 @@ message getReturnMsg {
...
 
 /*
-** GetBlockByHash (blockHash) => Block
-** GetBlockByHash receives a hash as an input and
+** getBlockByHash (blockHash) => Block
+** getBlockByHash receives a hash as an input and
 ** returns an RLP serialized byte[] Block
 */
 message getBlockByHashInputMsg {
   bytes blockHash = 1;
```
Comment (no2chem) [inline verif-store.proto:53]: maybe we need to define a 32byte type

**Review by no2chem — COMMENTED** [review]
shouldn't need the Java file?
also see comments

File: verif-store.proto (hunk 3 above)
Comment (SoujanyaPonnapalli) [reply verif-store.proto:53]: Do we want to define a message type with eight fixed32 (4 byte) types? We could instead check the message._size() while encoding/decoding data into/from a protobuf message.

**Pull request merged**